ext/storage_sources/gcp_store/test/test_gcp_connection.cpp

```diff
@@ -30,5 +30,266 @@
 #include <catch2/catch.hpp>
 
 #include "gcp_connection.h"
+#include <typeinfo>
 
-TEST_CASE("Testing class gcpConnection", "gcp-connection") {}
+namespace gcs = google::cloud::storage;
+using namespace gcs;
+
+static std::string
+create_file(const std::string file_name, const std::string payload)
+{
+    std::ofstream file(file_name);
+    file << payload;
+    file.close();
+    return file_name;
+}
+
+static auto
+upload_file(gcs::Client client, std::string bucket_name, const std::string bucket_prefix,
+  const std::string file_name, const std::string object_name)
+{
+    auto metadata = client.UploadFile("./" + file_name, bucket_name, bucket_prefix + object_name);
+
+    return metadata;
+}
+
+static bool
+check_exist(gcs::Client client, std::string bucket_name, const std::string bucket_prefix,
+  const std::string object_name)
+{
+    auto metadata = client.GetObjectMetadata(bucket_name, bucket_prefix + object_name);
+
+    // Metadata ok implies that the file is present.
+    if (metadata.ok()) {
```

Comment:
we can directly return `metadata.ok() `here

Reply:
addressed in [2ace538](https://github.com/wiredtiger/wiredtiger/pull/8749/commits/2ace538bb48cef22cdd59a28392f5f0ea56b8396)